src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCaller.java

```diff
@@ -174,6 +181,13 @@ protected String[] customCommandLineValidation() {
                     hcArgs.getDragenNameValuePairs(),
                     HaplotypeCallerArgumentCollection.DRAGEN_GATK_MODE_LONG_NAME);
         }
+        if (hcArgs.flowMode != HaplotypeCallerArgumentCollection.FlowMode.NONE) {
+            ModeArgumentUtils.setArgValues(
```

Comment:
I notice there is now some extra outputs related to the calling class in the mode argument utils command line output here. Can you fix this?  
```
WARN  ModeArgumentUtils - * The following arguments have had their inputs overwritten:            *
10:34:09.383 WARN  ModeArgumentUtils - * --apply-bqd true (java.io.PrintStream@33359276)                       *
10:34:09.383 WARN  ModeArgumentUtils - * --apply-frd true (java.io.PrintStream@7ca1d70b)
```

Reply:
Fixed